playbooks/roles/content_host/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+katello_organization: "Default_Organization"
+katello_environment: "Library"
+katello_username: "admin"
+katello_password: "changeme"
```

Comment:
Best practices in Ansible say to preface variables with the name of the role to avoid collisions. So this would then be, `content_host_katello_password`. I might suggest consider naming this role `katello_client` instead of `content_host` since I think that more accurately describes how we talk about a client machine.